assets/src/edit-story/elements/text/karma/edit.karma.js

```diff
@@ -136,4 +136,70 @@ describe('TextEdit integration', () => {
       });
     });
   });
+
+  describe('add a multiline text element', () => {
+    let element;
+    let frame;
+    let text = '\n\nThis is some test text.\n\nThis is more test text.\n\n';
+
+    beforeEach(async () => {
+      const insertElement = await fixture.renderHook(() => useInsertElement());
+      element = await fixture.act(() =>
+        insertElement('text', {
+          font: TEXT_ELEMENT_DEFAULT_FONT,
+          content: text,
+          x: 40,
+          y: 40,
+          width: 300,
+        })
+      );
+
+      frame = fixture.editor.canvas.framesLayer.frame(element.id).node;
+    });
+
+    it('should render initial content', () => {
+      expect(frame.textContent).toEqual(text);
+    });
+
+    describe('edit mode', () => {
+      let textElement;
+      let initialHeight;
+
+      beforeEach(async () => {
+        await fixture.events.click(frame);
+        textElement = fixture.querySelector('[data-testid="textEditor"]');
+        initialHeight = textElement.getBoundingClientRect()?.height;
+      });
+
+      it('should not change height when entering and exiting edit mode', async () => {
```

Comment:
Could we add a snapshot here both before and after entering edit mode and after leaving it again? Just to make sure it looks correctly too.

It's just inserting `await data.fixture.snapshot('Trailing and leading newlines, before editmode');` or similar description in three different places here (and confirming the snapshots on Percy).

Reply:
Done 👍